modules/apiserversource-yaml.adoc

```diff
@@ -145,15 +145,18 @@ spec:
 
 .Verification
 
-To verify that the Kubernetes events were sent to Knative, you can look at the message dumper function logs.
+To verify that the Kubernetes events were sent to Knative, you can look at the event-display logs or use web browser to see the events.
 
-. Get the pods by entering the following command:
+. To view the events in a web browser, open the link returned by the following command:
 +
 [source,terminal]
 ----
-$ oc get pods
+$ oc get ksvc event-display -o jsonpath='{.status.url}'
 ----
-. View the message dumper function logs for the pods by entering the following command:
+
```

Comment:
Need a `+` to keep the image in line with the text.
```suggestion
+
```
The missing `+` also affects the numbering. See [Verification in the preview](https://60772--docspreview.netlify.app/openshift-serverless/latest/eventing/event-sources/serverless-apiserversource#apiserversource-yaml_context).